kong/plugins/aws-lambda/handler.lua

```diff
@@ -1,39 +1,24 @@
 -- Copyright (C) Kong Inc.
 local BasePlugin = require "kong.plugins.base_plugin"
 local aws_v4 = require "kong.plugins.aws-lambda.v4"
-local singletons = require "kong.singletons"
-local responses = require "kong.tools.responses"
-local constants = require "kong.constants"
-local utils = require "kong.tools.utils"
-local meta = require "kong.meta"
 local http = require "resty.http"
 local cjson = require "cjson.safe"
```

Comment:
Both of those `http` and `cjson` will eventually go away too with `kong.json` and `kong.http` :rocket: